### ISSUE TITLE
Status page showing all modules

### ISSUE THREAD

**Issue by philipp2310** [project-alice-assistant/ProjectAlice#620]

**Comment by Psychokiller1888** [comment]
I don't like colors, I don't see them well, and so do many people. Yes, the skills use a color code, and that's already bad imo

**Comment by philipp2310** [comment]
yes, as I have a color weakness as well, I agree. The icons them selfes should reflect the status as well.
I was already taking a look at FA-Icons, but didn't find the perfect ones so far.

Same should be done for skills I guess. Would be as simple as adding in the shadow of an exclamation mark or an x in the colored corner.
One barrier easily removed, that we should keep in mind!
